packages/ppbgdi/data_stream/kubernetes_event_logs/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -148,6 +152,10 @@ processors:
       field: _tmp.regarding.resourceVersion
       target_field: kubernetes.event.regarding.version
       ignore_missing: true
+  - rename:
+      field: _tmp.regarding.reportingInstance
```

Comment:
what is instance referring to here? node or kubernetes object? or sth else?

Reply:
Seems to be the node. You may check in the logs. Problem is the logs are appear as they are, without much documentation to be found about it.